siliconcompiler/schema/schema_cfg.py

```diff
@@ -311,6 +311,19 @@ def schema_pdk(cfg, stackup='default'):
             optimization. Node examples include 180, 130, 90, 65, 45, 32, 22 14,
             10, 7, 5, 3.""")
 
+    scparam(cfg,['pdk', pdkname, 'lambda'],
```

Comment:
It feels like this is intended to be a design parameters, so should it be in `option,lambda` instead?